src/features/MovieList/usePopularMovies.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { url_back, url_front_movie } from "../../common/API/requests";
+
+const page = "1";
+const url = `${url_front_movie}popular${url_back}&page=${page}`;
+
+export const usePopularMovies = () => {
+    const [popularMovies, setPopularMovies] = useState({
+        status: "loading",
+        data: [],
```

Comment:
This element is unnecessary {data: [],).

Reply:
I think it is necessary, cause it doesn't always work without it and adding it resolved the errors